<?xml version="1.0"?>
<launch>

  <arg name="input_pointcloud_topic" default="/arm_cam3d/depth_registered/points" />
  <arg name="target_frame" default="odom" />
  <arg name="classifier" default="asus-nagoya" />

  <group ns="mir_perception">
    <node pkg="mir_object_recognition" type="pcl_object_recognizer_node" name="pcl_object_recognizer" output="screen" 
          respawn="false" ns="multimodal_object_recognition/recognizer/pcl">
      <param name="classifier" value="$(arg classifier)" type="str" />
      <remap from="~input/object_list" to="/mir_perception/multimodal_object_recognition/recognizer/pcl/input/object_list" />
      <remap from="~output/object_list" to="/mir_perception/multimodal_object_recognition/recognizer/pcl/output/object_list"/>
    </node>
  </group>

</launch>